bot/common/threads/onboarding.py

```diff
@@ -2,6 +2,7 @@
 import discord
 from bot.common.airtable import (
     find_user,
+    update_member,
```

Comment:
interesting... good call.  This is on if a user runs the /update command that we also update both the display name in the user table and membert table?

Reply:
Both in /update, and on /join. This airtable command is used in the join flow, since the user record is actually created without the display name, and then updated with it on a later step (see commands.py, line 143 for the initial user creation)